app/models/icon.rb

```diff
@@ -4,5 +4,4 @@ class Icon < ApplicationRecord
   default_scope -> { kept }
 
   belongs_to :portfolio_item
-  validates :data, :presence => true
```

Comment:
Will simply pulling this out cause issues with the json render?  I'm thinking the source will be populated with `nil`, was wondering if we shouldn't send empty data back in the case where `data` is nil?